CODEOWNERS

```diff
@@ -12,4 +12,5 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-* @pjanotti @sjkaris @rghetia @tedsuo @jmacd
+* @pjanotti @sjkaris @rghetia @tedsuo @jmacd @iredelmeier @paivagustavo @krnowak @lizthegrey
+exporter/trace/stackdriver/* @ymotongpoo
```

Comment:
Consider to remove this, we do not have a policy about partial ownership. Consider to add that to the community documents first.

Reply:
sure, one change at a time, I'll remove.